src/test/java/org/datacleaner/extension/xml/XPathTransformerTest.java

```diff
@@ -11,9 +11,14 @@
 import org.datacleaner.data.MockInputRow;
 import org.easymock.EasyMock;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 public class XPathTransformerTest {
+    @Rule
+    public final ExpectedException expectedException = ExpectedException.none();
```

Comment:
Rules! Beautiful! :)

(Getting @kaspersorensen to use these has been an uphill battle. :))